compiler/src/iree/compiler/Codegen/Common/GenericVectorization.cpp

```diff
@@ -289,6 +289,18 @@ void GenericVectorizationPass::runOnOperation() {
                             vectorizeGatherAccesses);
   };
 
+  {
+    // Canonicalize mask related ops before we lower them.
+    RewritePatternSet maskCanonPatterns(funcOp.getContext());
+    vector::CreateMaskOp::getCanonicalizationPatterns(maskCanonPatterns,
+                                                      funcOp.getContext());
+    vector::ConstantMaskOp::getCanonicalizationPatterns(maskCanonPatterns,
+                                                        funcOp.getContext());
+    vector::MaskOp::getCanonicalizationPatterns(maskCanonPatterns,
+                                                funcOp.getContext());
+    (void)applyPatternsAndFoldGreedily(funcOp, std::move(maskCanonPatterns));
```

Comment:
Could we catch the error and return it instead of ignoring it?

Reply:
done, just copy pasted from this file. It looks like we are ignoring the returns everywhere else

Reply:
ok, I didnt notice that. We shouldnt be doing that. If you are here already,  would be great if you could fix that everywhere (at least in this file)